components/cluster.py

```diff
@@ -111,30 +111,29 @@ def get_component(session_id):
     return html.Div(items)
 
 
-def build_cluster_component(mindate, maxdate, session_id, id="1"):
-
-    return html.Div([dcc.DatePickerRange(
-        id='date-pick-'+id,
-        min_date_allowed=mindate,
-        max_date_allowed=maxdate,
-    ),
+def build_cluster_component(mindate, maxdate, session_id, idd="1"):
+    id_component = "date-pick-{}".format(idd)
+    date_component = date_picker.get_component(
+        mindate, maxdate, mindate + datetime.timedelta(days=1), id_component)
```

Comment:
Currently, dates are automatically set, causing the clustering to start running on entering the view. If you pass `None` as the third argument, the selection will remain empty until the user selects something.

Reply:
Ok!